cloud-gcp/src/test/java/com/sequenceiq/cloudbreak/cloud/gcp/GCPLoadBalancerMetadataCollectorTest.java

```diff
@@ -226,9 +272,20 @@ private ForwardingRule createPrivateFowardingRule() {
         forwardingRule.setLoadBalancingScheme("INTERNAL");
         forwardingRule.setName(FORWARDING_RULE_NAME_2);
         forwardingRule.setIPAddress(PRIVATE_IP);
+        forwardingRule.setPorts(List.of(PORT2));
+        forwardingRule.setBackendService(BACKEND_SERVICE_NAME);
         return forwardingRule;
     }
 
```

Comment:
If you don't have a test case for when `ForwardingRule.getPorts()` is null, you should add one. That's what tripped me up in CB-14040.

Reply:
good idea!